### PR TITLE
feat: InventoryTracker and Invsee command

### PR DESCRIPTION
**Pull request by Zirada**
This module tracks the inventories of other players by storing their held items along with the timestamp. The tracked inventory can then be viewed with the invsee command. I also added two new events: EntityEquipmentChangeEvent, which is called when an EntityEquipmentUpdateS2CPacket is received and LoreQueryEvent, which is called when the tooltip for an item is requested.

### CODE REVIEW

**Comment by ghost** [comment]
Avoid using Overwrite in Mixins.

**Comment by Zirada** [comment]
Alright. I just used it because I didn't know how to inject into a lambda but I found a better solution now.

File: src/main/kotlin/net/ccbluex/liquidbounce/event/events/GameEvents.kt [hunk 1]
```diff
@@ -112,3 +113,9 @@ class OverlayMessageEvent(val text: Text, val tinted: Boolean) : Event()
 
 @Nameable("perspective")
 class PerspectiveEvent(var perspective: Perspective) : Event()
+
+class LoreQueryEvent(val itemStack: ItemStack, val lore: ArrayList<Text>) : Event() {
+    fun addLore(text: String?) {
+        lore.add(Text.of(text))
+    }
+}
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/event/events/GameEvents.kt]: Put `@Nameable` as well. Might be interesting for ScriptAPI folks as well.

**Review by 1zun4 — APPROVED** [review]
Code looks good and good idea. I still have to try it out though.

**Pull request merged**